menas-web/ui/index.html

```diff
@@ -0,0 +1,120 @@
+<!--
+  ~ Copyright 2018 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE HTML>
+
+<html>
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta http-equiv='Content-Type' content='text/html;charset=UTF-8'/>
+
+    <script src="node_modules/@openui5/sap.ui.core/src/sap-ui-core.js" id="sap-ui-bootstrap"
+            data-sap-ui-compatVersion="edge"
+            data-sap-ui-libs="sap.m,sap.ui.table,sap.ui.layout,sap.tnt"
+            data-sap-ui-bindingSyntax="complex"
+            data-sap-ui-resourceroots='{
+            "sap.m": "../node_modules/@openui5/sap.m/src/sap/m",
+            "sap.ui": "../node_modules/@openui5/sap.ui.core/src/sap/ui",
+            "sap.tnt": "../node_modules/@openui5/sap.tnt/src/sap/tnt",
+            "sap.ui.table": "../node_modules/@openui5/sap.ui.table/src/sap/ui/table",
+            "sap.ui.layout": "../node_modules/@openui5/sap.ui.layout/src/sap/ui/layout",
+            "sap.ui.unified": "../node_modules/@openui5/sap.ui.unified/src/sap/ui/unified"
+            }'
+            data-sap-ui-theme='sap_belize'
+            data-sap-ui-theme-roots='{
+              "sap_belize": "theme"
+            }'></script>
+
+    <script src="node_modules/moment/min/moment.min.js"></script>
+    <script src="node_modules/lodash/lodash.js"></script>
+    <script src="node_modules/cronstrue/dist/cronstrue.js"></script>
+
+    <script>
+        function _menasLoadElem(sElemType, sSrc, aAttrs, fnOnLoad) {
+            const el = document.createElement(sElemType);
+            const srcAttr = sElemType === "link" ? "href" : "src";
+            el[srcAttr] = `${sSrc}`;
+            aAttrs.forEach((attr) => {
+                el.setAttribute(attr.k, attr.v);
+            })
+            if (fnOnLoad) {
+                el.addEventListener("load", fnOnLoad);
+            }
+            document.getElementsByTagName("head").item(0).append(el);
+        }
+
+        $.getJSON('package.json', function(json) {
+            window.apiUrl = json.config.apiUrl;
+            window.lineageReadApiUrl = json.config.lineageReadApiUrl;
```

Comment:
**Proposition:** I wouldn't bother with Lineage/Spline now. Will see how it be integrated.

Reply:
If Spline is not working we cannot merge this. Spline on-prem still needs to work